packages/remix-server-runtime/__tests__/server-test.ts

```diff
@@ -1679,4 +1679,37 @@ describe("shared server runtime", () => {
       expect(spy.console.mock.calls.length).toBe(1);
     });
   });
+
+  describe("header requests", () => {
+    test.only("can access loader headers if parent doesn't have a loader itself", async () => {
```

Comment:
I think you forgot to remove this `only`.

Reply:
Nice catch, fixed. Thanks very much for commenting 👍